tests/src/rules/dynamic-import-chunkname.js

```diff
@@ -412,7 +412,7 @@ ruleTester.run('dynamic-import-chunkname', rule, {
         /* webpackPrefetch: true */
         /* webpackPreload: true */
         /* webpackIgnore: false */
-        /* webpackMode: "eager" */
+        /* webpackMode: "lazy" */
```

Comment:
This test isn't checking for eager mode functionality, it's checking whether other modes are valid, so changing this to "lazy" so it doesn't interfere with the eager mode conditions.